CHANGELOG.md

```diff
@@ -1,48 +1,61 @@
-* New: Public and private profiles do not have the same design ([#325](https://github.com/GlotPress/GlotPress-WP/issues/325))
-* New: Translation Sets: Validate slugs on saving ([#329](https://github.com/GlotPress/GlotPress-WP/issues/329))
-* New: Make translation sets deletable ([#267](https://github.com/GlotPress/GlotPress-WP/issues/267))
-* Bugfix: Mark Sindhi locale as RTL ([#243](https://github.com/GlotPress/GlotPress-WP/issues/243))
-* Bugfix: Replace `current_user_can( 'manage_options' )` with GP permissions ([#254](https://github.com/GlotPress/GlotPress-WP/issues/254))
-* Bugfix: Child projects are inaccessible if permalinks have trailing slash ([#265](https://github.com/GlotPress/GlotPress-WP/issues/265))
-* Bugfix: Various code cleanups to improve code quality ([#237](https://github.com/GlotPress/GlotPress-WP/issues/237))
-* Bugfix: Update readme files ([#228](https://github.com/GlotPress/GlotPress-WP/issues/228))
-* Bugfix: Replace deprecated javascript back code ([#278](https://github.com/GlotPress/GlotPress-WP/issues/278))
-* Bugfix: `[_]wp_specialchars()` is deprecated/private and should be replaced ([#280](https://github.com/GlotPress/GlotPress-WP/issues/280))
-* Bugfix: Merge similar translation strings and avoid using HTML tags in translation strings ([#295](https://github.com/GlotPress/GlotPress-WP/pull/295))
-* Bugfix: Actions for translations have no `gp_` prefix ([#232](https://github.com/GlotPress/GlotPress-WP/issues/232))
-* Bugfix: `$original->validate()` fails if singular is '0' ([#301](https://github.com/GlotPress/GlotPress-WP/issues/301))
-* Bugfix: Auto generation of a project slug doesn't work correctly in some cases ([#328](https://github.com/GlotPress/GlotPress-WP/issues/328))
-* Bugfix: Suspend cache invalidation during original imports ([#332](https://github.com/GlotPress/GlotPress-WP/issues/332))
-* Bugfix: Do not allow translations with empty plurals ([#308](https://github.com/GlotPress/GlotPress-WP/pull/308))
-* Bugfix: Update schema definitions to work with WP's `dbDelta()` ([#343](https://github.com/GlotPress/GlotPress-WP/issues/343))
-* Bugfix: Translation Set: Use the correct select helper for the edit form ([#351](https://github.com/GlotPress/GlotPress-WP/pull/351))
-* Bugfix: Translation Sets: A failed update redirects to /sets/-new ([#349](https://github.com/GlotPress/GlotPress-WP/pull/349))
-
-* BREAKING: Don't store counts as a public property of GP_Translation_Set ([#334](https://github.com/GlotPress/GlotPress-WP/issues/334))
-* Bugfix: Prevent a PHP fatal error when importing originals ([#302](https://github.com/GlotPress/GlotPress-WP/pull/302))
-
-Thanks to all the contributors so far: Aki Bj�rklund, Daisuke Takahashi, Dominik Schilling, Gabor Javorszky, Greg Ross, Peter Dave Hello, Rami, Sergey Biryukov
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## 2.0.0-beta.2 (March 27, 2016)
+
+### Security
```

Comment:
Just to note that when the changelog get's added to the readme.txt file during release, these `###` sections will remain as they are, the readme.txt standard doesn't have any equivalent.